QgisModelBaker/libqgsprojectgen/dbconnector/gpkg_connector.py

```diff
@@ -172,6 +172,19 @@ def _get_tables_info(self):
         cursor.close()
         return complete_records
 
+    def get_meta_attrs_info(self):
```

Comment:
Can you add this method to the base class with a documentation comment?
The content of the class can be `raise NotImplementedError()`